src/App.jsx

```diff
@@ -14,33 +16,81 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = 'alphabetSort';
+const SORT_BY_LENGTH = 'goodsLength';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function sortGoods(goods, { sortField, isReversed }) {
+  const newSortGoogs = [...goods];
```

Comment:
There seems to be a typo in the variable name `newSortGoogs`. It should probably be `newSortedGoods` to better reflect its purpose.